utils/nvim/setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This change doesn't seem related to user snippets. Am I missing something?

Same question about `utils/treesitter/.gitignore`.

Reply:
With `/bin/sh` operations like `set` doesn't work